captum/attr/_core/deep_lift.py

```diff
@@ -792,7 +950,14 @@ def maxpool3d(module, inputs, outputs, grad_input, grad_output, eps=1e-10):
 
 
 def maxpool(
-    module, pool_func, unpool_func, inputs, outputs, grad_input, grad_output, eps=1e-10,
+    module: Module,
+    pool_func: Callable,
+    unpool_func: Callable,
+    inputs,
+    outputs,
+    grad_input,
+    grad_output,
+    eps: float = 1e-10,
 ):
```

Comment:
This method has two instances of `list(module.input.shape)`, you can replace these with `list(cast(torch.Size, module.input.shape))`